src/tests/functional/plugin/shared/src/snippets/mha.cpp

```diff
@@ -66,6 +69,12 @@ void MHA::SetUp() {
         rel_threshold = 0.05f;
 }
 
+void MHA::compile_model() {
+    if (m_thread_count > 0)
```

Comment:
Can we introduce some static variable for default thread count? E.g. `default_thread_count`
I believe that this could improve test instances readability. m_thread_count=0 looks a bit confusing in test instances at first glance.

Reply:
Oh, now I see your point. Thanks a lot for the detailed explanation! I agree, let's add.